src/components/networked.js

```diff
@@ -42,7 +42,7 @@ AFRAME.registerComponent('networked', {
 
     this.conversionEuler = new THREE.Euler();
     this.conversionEuler.order = "YXZ";
-    this.interpolationBuffers = [];
+    this.bufferInfos = [];
```

Comment:
This isn't really your fault or anything, but reading this code again makes me question this design choice. It seems to me like there are several possibilities for where to stick the per-entity `InterpolationBuffer`s:

1. One { entity -> buffer } mapping of them for the browser window.
2. One { entity -> buffer } mapping per networked scene.
3. One { entity -> buffer } mapping per networked component.
4. One buffer on the user data of each entity's `Object3D`.

We picked choice 3, but it seems kind of arbitrary. I might have done choice 4 or choice 2.

Reply:
The only risk I can think of with 4 is that the user data of the `Object3D` is a shared namespace. It may be an unnecessary (albeit small) risk of name conflicts.
The rest seem fine. The map is very small in choice 3, whereas the map is proportional to the size of the scene (or scenes) in choice 3 or 1. Since the "map" is actually an array that we traverse linearly, we might care about its size. Then again, it may be wrong to say we're traversing it "linearly" because although `bufferInfos` _is_ a js array, we end up following object pointers around at each step.
Anyway, I'm not too concerned with where this data lives. Choice 3 seems fine to me but let's keep this in mind if there's a reason to move it.